docs/CLI-era-command-groups.md

```diff
@@ -0,0 +1,27 @@
+`cardano-cli` is transitioning to a new command structure where instead of each
+era-senstivie command getting an era flag (eg `--babbage-era`, `--conway-era`),
+we will have the era as a top-level command.
+
+For the moment, this will be:
+
+* `byron`: already exists and will be unchanged
+* `shelley`: re-purposed to mean `shelley-era` only rather than `shelley-based` era
+* `allegra`: `allegra` era only
+* `mary`: `mary` era only
+* `alonzo`: `alonzo` era only
+* `babbage`: `babbage` era only
+* `conway`: `conway` era only
+
+Then we will have the following:
+
+* `latest`: the latest supported era.  The latest officially supported era in the CLI
+* `experimental`: the era following the latest era that is not yet officially supported in the CLI
```

Comment:
We've removed experimental and I'm not 100% sure on latest. The latest should just be the default without specifying an era.

Reply:
For eras that are experimental we can update the help text to say this.